src/libraries/Microsoft.VisualBasic.Core/src/Resources/Strings.resx

```diff
@@ -116,285 +116,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="ID3" xml:space="preserve">
-    <value>This Error number is obsolete and no longer used.</value>
-  </data>
-  <data name="ID5" xml:space="preserve">
-    <value>Procedure call or argument is not valid.</value>
-  </data>
-  <data name="ID6" xml:space="preserve">
```

Comment:
>ID6 [](start = 14, length = 3)

It looks like a number of the "ID#" resources may be used for exception messages from `ErrObject`. For instance ID6, ID9, ID11, ID13, ...

See `ErrObject.MapExceptionToNumber()` and `ErrObject.FilterDefaultMessage()`.

Reply:
Thanks, I will revert all IDs